charts/super-agent/values.yaml

```diff
@@ -19,6 +19,15 @@ super-agent-deployment:
   # -- In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located. Can be configured also with `global.customSecretLicenseKey`
   customSecretLicenseKey: ""
 
+  # -- L1 identity client_id to use. Can be configured also with `global.l1IdentityClientId`
+  l1IdentityClientId: ""
+  # -- L1 identity client_secret to use. Can be configured also with `global.l1IdentityClientSecret`
+  l1IdentityClientSecret: ""
+  # -- In case you don't want to have the client_id and client_secret in you values, this allows you to point to a user created secret to get the key from there. Can be configured also with `global.customL1IdentityClientSecretKey`
+  customL1IdentitySecretName: ""
+  customL1IdentityClientIdSecretKey: ""
+  customL1IdentityClientSecretSecretKey: ""
```

Comment:
below we have already:
```
      auth:
        # -- Organization ID where fleets will live.
        organizationId: ""
        secret:
          create: true
          # -- Name auth' secret provided by the user. If the creation of this secret is set to `true`, this is the same the secret
          # will have.
          # @default -- release name suffixed with "-auth"
          name:
          private_key:
            # -- Key inside the secret containing the private key.
            # @default -- `private_key`
            secret_key:
            # -- In case `.config.auth.secret.create` is true, you can set these keys to set private key directly as base64.
            # This options is mutually exclusive with `plain_pem`.
            base64_pem:
            # -- In case `.config.auth.secret.create` is true, you can set these keys to set private key directly as plain text.
            # This options is mutually exclusive with `base64_pem`.
            plain_pem:
          client_id:
```

why it is not in the auth section?

Reply:
is it deprecated such section? if so we should mark it accordingly

Reply:
The client_id in that section is the L2 client_id, and I thought that it was used for configuration. The other ones are in the same level of userKey, that are only used for L1 creation. I can move it but I'm not sure as userKey was already out of this section.

Quick sync discussion?